Cargo.toml

```diff
@@ -42,6 +42,10 @@ os-ext = [
 # Enables `mio::net` module containing networking primitives.
 net = []
 
+# Forces the use of the poll system call instead of epoll on systems
+# where both are available
+force-old-poll = ["os-poll"]
```

Comment:
We're not going to add a feature for this or support `poll` on any OS that has something better (e.g. `epoll` or `kqueue`). Mio is complex enough as-is it maintain with the matrix OSs architectures we're not getting selectors in the mix as well.

Reply:
This will also be removed once this PR is ready, I currently use this to develop the `poll` selector on Linux because development on Haiku and the espidf is quite hard. I still run the tests for Haiku, but this allows me to iron out bugs on a system which is easy to develop for.

TL;DR: Will be removed when PR is ready